### ISSUE TITLE
Add "half time"

### ISSUE THREAD

**Issue by qreeves** [red-eclipse/base#374]
For mode/map combinations that require teams to switch sides, a small break for "half time" could be added to reinforce the fact that the setup is changing.

**Comment by freezurbern** [comment]
Also possibly include announcer yelling Halftime! or similar.

**Comment by Calinou** [comment]
> Also possibly include announcer yelling Halftime! or similar.

I think it's more consistent to have "Five minutes remaining" announcer in all modes and maps.

**Comment by bonifarz** [comment]
I agree a short timer and/or announcement will make the team change less abrupt.
"Five minutes remaining" sounds cool, unless `/timelimit` is altered. So, I guess a generic term would be better.

**Comment by SadisticMystic** [comment]
Multi exists too, and when there's mapbalance in that, players will have to get resituated for the first and third quarters as well.

**Comment by shirepirate** [comment]
i think the announcer fleshing things out for better understanding is a good way to handle it, better than increasing the length of the match anyway. doesn't have to be complicated:

announcer: "team switch in 10 seconds"
announcer: "teams switched"

as for multi, there's only 1 multi map in the rotation as far as i know that has full map balance currently (discontinuity)

**Comment by shirepirate** [comment]
i think i've changed my opinion of this, it would be good to have a 10 second break before sides are switched in any circumstance. would need to be made clear in some way of course. if @qreeves and his family record more sounds before the next release we can choose what the announcer will say.
